xl_auth/settings.py

```diff
@@ -11,6 +11,8 @@
 class Config(object):
     """Base configuration."""
 
+    SERVER_NAME = os.environ.get('SERVER_NAME', None)
+    PREFERRED_URL_SCHEME = os.environ.get('PREFERRED_URL_SCHEME', 'http')
```

Comment:
I was thinking, maybe it would be nice with a check somewhere that ensures we don't accidentally use `http` instead of `https` in production? Nothing to fix now, but maybe add an issue?

Reply:
Terminated by Nginx! Only used for generating URLs.